src/components/cuisine/Cuisine.js

```diff
@@ -1,39 +1,78 @@
 import React, { Component } from 'react'
 import BackButton from '../common/BackButton'
```

Comment:
may want to consider using absolute imports so that it'll be easier to handle if u want to change the file structure in the future?

https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

also i think we should put all the user-form components under a sub folder

Reply:
agree about the sub folder, i was going to suggest grouping components by feature as opposed to what i initially said (all components on same level)

Reply:
and yeah absolute imports definitely